rtos/nuttx/wolfssl/README.md

```diff
@@ -0,0 +1,78 @@
+# NuttX + wolfSSL - Proof-of-Concept / Preview
+
+## Instructions
+
+1) Create working directory (e.g. ~/nuttxspace):
+    ```
+    $ cd ~
+    $ mkdir nuttxspace
+    ```
+2) Install dependencies:
+    ```
+    $ cd ~/nuttxspace
+    $ sudo su
+    $ apt install -y bison flex gettext texinfo libncurses5-dev libncursesw5-dev gperf automake libtool pkg-config build-essential gperf genromfs libgmp-dev libmpc-dev libmpfr-dev libisl-dev binutils-dev libelf-dev libexpat-dev gcc-multilib g++-multilib picocom u-boot-tools util-linux
+    $ apt install -y kconfig-frontends
+    $ apt install -y gcc-arm-none-eabi binutils-arm-none-eabi
+    $ exit # exit sudo
+    ```
+3) Clone nuttx and nuttx-apps into working directory:
+    ```
+    $ git clone https://github.com/apache/nuttx.git nuttx
+    $ git clone https://github.com/apache/nuttx-apps apps
+    ```
+4) Copy this directory reanamed to "wolfssl" into the working directory applications:
+    ```
+    $ cd <path to where this archive was extracted>
+    $ cp -r wolfSSL-nuttX-app ~/nuttxspace/apps/crypto/wolfssl
+    ```
+5) Setup wolfSSL in preparation for the build:
+    ```
+    $ cd ~/nuttxspace/apps/crypto/wolfssl
+    $ ./setup-wolfssl.sh
+    ```
+> **NOTE:** There is a conflict with the function `mutex_test()` in `wolfcrypt/test/test.c`
```

Comment:
Is this patch required still? Consider removing this comment.

Reply:
no patch is no longer required, I'll remove